packages/pigment-css-react/src/baseAtomics.js

```diff
@@ -0,0 +1,43 @@
+import { generateAtomics } from './generateAtomics';
+
+export const stackAtomics = generateAtomics(({ theme }) => ({
```

Comment:
I wonder why `stockAtomics` is not declared in `Stack.jsx`, is it intended to be used elsewhere?

Reply:
Having the atomics generator on the component file makes sense to me as well 👍🏼